pwndbg/color/__init__.py

```diff
@@ -73,3 +77,59 @@ def terminateWith(x, color):
 
 def ljust_colored(x, length, char=' '):
     return x + (length - len(strip(x))) * char
+
+
+try:
+    import pygments
+    import pygments.lexers
+    import pygments.formatters
+    from pwndbg.color.lexer import PwntoolsLexer
+except ImportError:
+    pygments = None
+
+pwndbg.config.Parameter('syntax-highlight', True, 'Source code / assembly syntax highlight')
+style = theme.Parameter('syntax-highlight-style', 'monokai', 'Source code / assembly syntax highlight stylename of pygments module')
+
+@pwndbg.config.Trigger([style])
+def check_style():
+    try:
+        formatter = pygments.formatters.Terminal256Formatter(
+            style = str(style)
+        )
+    except pygments.util.ClassNotFound:
+        print("The pygment formatter style '%s' is not found, restore to default" % style)
```

Comment:
This should use the new ```pwndbg.color.message``` module's ```warn()``` method

Reply:
Fixed